cmd/thor/flags.go

```diff
@@ -151,6 +151,12 @@ var (
 		Usage: "metrics service listening address",
 	}
 
+	allowedTracersFlag = cli.StringFlag{
+		Name:  "allowed-tracers",
+		Value: "all",
```

Comment:
Should this be `"*"` to be consistent with cors?

Reply:
Doesn't matter I think.. I believe that in CORS the `*` is a regex parser.
In the `api-allowed-tracers` it's just a keyword.
Wdyt ?